modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/checkpoint/Checkpointer.java

```diff
@@ -983,4 +986,29 @@ public CheckpointProgress currentProgress() {
     private boolean isShutdownNow() {
         return shutdownNow;
     }
+
+    /**
+     * Writes checkpoint performance statistics.
+     *
+     * @param chp Checkpoint.
+     * @param tracker CheckpointMetricsTracker.
+     */
+    private void writeStatistics(Checkpoint chp, CheckpointMetricsTracker tracker) {
```

Comment:
Can we mode this to `updateMetrics` method?

Reply:
done